annot8-components-print/src/main/java/io/annot8/components/print/processors/PrintSummarySettings.java

```diff
@@ -0,0 +1,19 @@
+/* Annot8 (annot8.io) - Licensed under Apache-2.0. */
+package io.annot8.components.print.processors;
+
+import io.annot8.api.settings.Description;
+
+public class PrintSummarySettings extends PrintSettings {
+
+  private int reportProgress = 100;
+
+  public PrintSettings setReportProgress(int reportProgress) {
+    this.reportProgress = reportProgress;
+    return this;
+  }
+
+  @Description("How often to report progress, set to 0 to disable")
```

Comment:
Worth explicitly stating the unit here? Is it every 100 items? Annotations? Seconds?